File: bootstrap/src/test/java/net/dontdrinkandroot/wicket/bootstrap/component/modal/SimpleModal.java
package net.dontdrinkandroot.wicket.bootstrap.component.modal;

import org.apache.wicket.model.IModel;
import org.apache.wicket.model.Model;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class SimpleModal extends Modal<Void>
{
    public SimpleModal(String id)
    {
        super(id);
    }

    @Override
    protected IModel<String> createHeadingModel()
    {
        return Model.of("Modal Heading");
    }
}

File: bootstrap/src/test/kotlin/net/dontdrinkandroot/wicket/bootstrap/component/modal/SimpleModal.kt
package net.dontdrinkandroot.wicket.bootstrap.component.modal

import org.apache.wicket.model.IModel
import org.apache.wicket.model.Model

class SimpleModal(id: String) : Modal<Void>(id) {

    override fun createHeadingModel(): IModel<String> {
        return Model.of("Modal Heading")
    }
}